src/ui_components/header/header.scss

```diff
@@ -1,7 +1,7 @@
 .header {
-  background: $access-green;
-  color: $off-white;
-  padding: 10px;
+  background: #1C1F2a;
```

Comment:
once you've defined $onelogin-black in the colors.scss file you should use the value here.

for example

.header {
  background: $onelogin-black;
}

Reply:
done